tests/playground.html

```diff
@@ -472,13 +472,6 @@
           </shadow>
         </value>
       </block>
-      <block type="math_change">
```

Comment:
Can you replicate this change in tests/generators, demo/code and other demo files.